lib/ansible/module_utils/network/routeros/routeros.py

```diff
@@ -71,6 +71,10 @@ def get_capabilities(module):
     return module._routeros_capabilities
 
 
+def check_args(module, warnings):
```

Comment:
This function is present for platform modules for sake of legacy reasons and not required anymore. If this function is not used anywhere in module code it can be removed.

Reply:
Oh, that's great news, thank you!